Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py

```diff
@@ -33,3 +35,274 @@ def test_get_members(mocker):
     mocker.patch('AWSGuardDuty.aws_session', return_value=client)
     members_res = get_members({})
     assert members_res['Contents'] == [{'AccountId': 1, 'DetectorId': 1, 'MasterId': 1}]
+
+
+def test_parse_incident_from_finding():
+    """
+    Given:
+    - Amazon GuardDuty finding with datetime object nested in it
+
+    When:
+    - Parsing finding to incident
+
+    Then:
+    - Ensure finding is parsed as expected
+    """
+    title = 'title'
+    desc = 'desc'
+    finding = {
```

Comment:
can we minimize the response to relevant fields only? should be focused on the relevant fields.
also, add an assert to check the date value.

Reply:
Yes

Reply:
keeping the whole finding for testing the the func better
adding assertion of the date parse